src/Yoti/Entity/Receipt.php

```diff
@@ -104,6 +106,14 @@ public function parseAttribute($attributeName, $pem)
         );
     }
 
+    /**
+     * @return \Yoti\Entity\ExtraData
+     */
+    public function getExtraData()
+    {
+        return ExtraDataConverter::convertValue($this->getAttribute(self::ATTR_EXTRA_DATA_CONTENT));
+    }
```

Comment:
Again, isn't this susceptible to also being parsed multiple times?  I know it's unlikely for `getExtraData` to be called more than once, but I think it's better to maybe to do the parsing up front once?

Reply:
Renamed to `parseExtraData` in bc018e8 (this is only called once when creating activity details)